check/all

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Runs multiple checks.
+#
+# The default set of checks most closely matches the tests run during
+# a continuous integration test, however these do not match perfectly
+# since those tests run on different architectures and environments.
+#
+# Defaults to the following tests:
+#   pylint
+#   mypy
+#   format-incremental
+#   pytest-and-incremental-coverage
+#   doctest
+#
+# Usage:
+#     check/all revision [BASE_REVISION] [--only-changed-files] [--apply-format]
```

Comment:
Should [--apply-format] be [--apply-format-changes]?

Reply:
fixed